### ISSUE TITLE
We need to allow for versioning of the master calibration frames

### ISSUE THREAD

**Issue by ndrory** [sdss/lvmdrp#90]
Title says it all ...

**Comment by havok2063** [comment]
I agree with this.  This will require some directory reorg on the SAS, along with updates to the `sdss-tree` product.  Then updates to the `lvmdrp` code where necessary.   We should decide on a versioning syntax, like a version tag number.  

For now, we can have manual backups on the SAS, but we might want to look into eventually putting the data into a version control product.  Not git though.

**Comment by ajmejia** [comment]
Add header keywords for versions of the pipeline and cals in calibrations and science products.

**Comment by ajmejia** [comment]
A reminder to myself about what we talked in the meeting today (also a minor update):

- The calibrations will behave like their own product, with their own version
- We propagate the version of the DRP that produced the calibrations in the header metadata

The path at SAS would be something like:

somwhere/in/sas/calib/{CALVER}/

So, no need for DRPVER in the path. I'll post an update on the file naming conventions once I get there.
